clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -792,35 +792,34 @@ message FilterResult {
     map<string, CollectorResult> nestedCollectorResults = 2;
 }
 
-// Specify how to highlight matched text in SearchRequest
+// Specify how to highlight matched text in SearchRequest.
 message Highlight {
 
-    enum Type {
-        DEFAULT = 0;
-        FAST_VECTOR = 1;
-    }
-
-    message Settings {
-        // Specify type of highlighter to use. Ignored right now in nrtsearch.
-        Type highlighter_type = 1;
-        // Used along with post_tags to specify how to wrap the highlighted text
-        repeated string pre_tags = 2;
-        // Used along with pre_tags to specify how to wrap the highlighted text
-        repeated string post_tags = 3;
-        // Number of characters in highlighted fragment, 100 by default
-        google.protobuf.UInt32Value fragment_size = 4;
-        // Maximum number of highlight fragments to return, 5 by default. If set to 0 returns entire text as a single fragment ignoring fragment_size.
-        google.protobuf.UInt32Value max_number_of_fragments = 5;
-        // Specify a query here if highlighting is desired against a different query than the search query
-        Query highlight_query = 6;
-        // Set to true to highlight fields only if specified in the search query
-        bool field_match = 7;
-    }
-
-    // Highlight settings
-    Settings settings = 1;
-    // Fields to highlight
-    repeated string fields = 2;
```

Comment:
Can you please revert these changes? These fields are already in use and modifying them will break the clients. Additionally please follow the [standard protobuf style](https://developers.google.com/protocol-buffers/docs/style) for new fields in this message which specifies using `underscore_separated_names` for field names. We have not followed the standard style in the past but want to do it for new proto objects.